src/stories/components/MainContent/styles.ts

```diff
@@ -0,0 +1,10 @@
+import { styled } from '@mui/material/styles';
+import { spacing } from 'theme/spacing';
+
+export const MainContentWrapper = styled('div')`
+  max-width: ${spacing.S1200};
+  min-height: calc(100vh - ${spacing.S269});
```

Comment:
Como eu falei em outro comentário, os `spacings` são para espaçamento e não para tamanho, exatamente por esse motivo, se a gente começar a criar `spacing` para todos os tamanhos vamos acabar criando inúmeros espaçamentos e não há necessidade. 
Sugiro colocar o tamanho direto em `rem` mesmo. Pode colocar o valor referente a 250px, vc acabou esquecendo de calcular o `padding-top` de 10px que tem no footer, isso faz ele aumentar 10px na altura dele